lib/DocStore.js

```diff
@@ -8,6 +8,7 @@ util.inherits( DocStore, Database );
 
 DocStore.prototype.reset = function(){
   this.db.exec('DROP TABLE IF EXISTS docs');
+  this.db.exec('DROP TABLE IF EXISTS rtree');
```

Comment:
I've also added this `DROP TABLE` statement to avoid `SqliteError: UNIQUE constraint failed: rtree.id` errors.

The triggers should really take care of this, but in the case where they don't this is more robust.